templates/settings.yml.erb

```diff
@@ -81,9 +81,14 @@
 # shared options for virsh DNS/DHCP provider
 :virsh_network: <%= scope.lookupvar("foreman_proxy::virsh_network") %>
 
-# Where our proxy log files are stored
-# filename or STDOUT
+# Log configuration
+# Uncomment and modify if you want to change the location of the log file or use STDOUT or SYSLOG values
 :log_file: <%= scope.lookupvar("foreman_proxy::log") %>
-# valid options are
-# WARN, DEBUG, Error, Fatal, INFO, UNKNOWN
+# Uncomment and modify if you want to change the log level
+# WARN, DEBUG, ERROR, FATAL, INFO, UNKNOWN
 :log_level: <%= scope.lookupvar("foreman_proxy::log_level") %>
+
+# Log buffer size and extra buffer size (for errors). Defaults to 3000 messages in total,
+# which is about 500 kB request.
+:log_buffer: <%= scope.lookupvar("foreman_proxy::log_buffer") %>
+:log_buffer_errors: <%= scope.lookupvar("foreman_proxy::log_buffer_errors") %>
```

Comment:
Could you check https://github.com/theforeman/smart-proxy/pull/368 please (which has already been merged)?  The default in the code appears to be 2000 + 1000, and it was only the config file comment that differed.